app/routes/admin/content/system-images.js

```diff
@@ -1,9 +1,59 @@
 import Ember from 'ember';
+import { eventTopics } from 'open-event-frontend/utils/dictionary/event';
 
 const { Route } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.i18n.t('System Images');
+  },
+
+  model() {
+    let arr = [];
+    const responseFromServer = [
+      {
+        name      : 'Air',
+        url       : 'http://lorempixel.com/400/200/sports/1/',
+        copyright : 'All rights reserved',
+        origin    : 'Google Images'
+      },
+      {
+        name      : 'Boat',
+        url       : 'http://lorempixel.com/400/200/sports/1/',
+        copyright : 'All rights reserved',
+        origin    : 'Google Images'
+      },
+      {
+        name      : 'Science',
+        url       : 'http://lorempixel.com/400/200/sports/1/',
+        copyright : 'All rights reserved',
+        origin    : 'Google Images'
+      },
+      {
+        name      : 'Yoga',
+        url       : 'http://lorempixel.com/400/200/sports/1/',
+        copyright : 'All rights reserved',
+        origin    : 'Google Images'
+      }
+
+    ];
+    for (var property in eventTopics) {
```

Comment:
Just add to the original response from the server. Don't replace it completely.


And also, try to make you of [lodash](https://lodash.com). We can use [flatMap](https://lodash.com/docs/4.17.4#flatMap) & [difference](https://lodash.com/docs/4.17.4#difference).

The following nested loops could be completely replaced by

```js
const missingSubTopics = difference(
	flatMap(eventTopics),
	flatMap(responseFromServer, 'name')
).map(name => {
	return {
		name,
		url: '',
		copyright: '',
		origin: ''
	}
});
responseFromServer.push(...missingSubTopics);
``` 

Now, you can just return `responseFromServer` as the model